build.sbt

```diff
@@ -14,7 +14,8 @@ incOptions := incOptions.value.withNameHashing(true)
 
 libraryDependencies ++= Seq(
     "com.kinja" %% "amqp-client" % "1.5.0",
-    "com.typesafe.play" %% "play-json" % "2.3.4"
+    "com.typesafe.play" %% "play-json" % "2.3.4",
+    "com.typesafe.slick" %% "slick" % "1.0.1"
```

Comment:
This way profile team won't be able to use it as they are using slick 2 i guess, and notification, which is using slick 3.
We should either maintain different branches for different slick versions, or put the mysql data store to a separate repository, and we'll have to deal with branches only there. This way the service can import the proper version of the messagestore dependency.
Both solutions are pain in the ass, but the seconds seems smaller to me, as we probably will change the datastore less frequently.